examples/datadog/main.tf

```
@@ -125,5 +125,7 @@ module "datadog" {
 
   cluster_name = module.k8s_platform.eks.cluster_name
 
+  datadog_secret = "dai-datadog/tamedia/keys"
```

```suggestion
  datadog_secret = "dai/datadog/tamedia/keys"
```
from https://tamedia.atlassian.net/wiki/spaces/PTS/pages/44304906/Secrets+Naming+convention+and+tags we should use "/"

---

I've created a second secret the old one is marked as obsolete and will be deleted once conformed the new one works.